gapis/api/gles/compat.go

```diff
@@ -601,12 +601,20 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				return
 			}
 		case *GlTexStorage2DMultisample:
-			{
+			if version.IsES || version.AtLeastGL(4, 3) {
+				// glTexStorage2DMultisample is supported by replay device.
 				cmd := *cmd
 				textureCompat.convertFormat(ctx, cmd.Target, &cmd.Internalformat, nil, nil, out, id, &cmd)
 				out.MutateAndWrite(ctx, id, &cmd)
-				return
+			} else {
+				// glTexStorage2DMultisample is not supported by replay device.
+				// Drop down to a non-multisampled version.
+				// TODO: Show report warning.
```

Comment:
Drop last two comments.
It is not downgrading to signle-sample.
I do not see a need to warn.  It is perfectly valid compat.

Spec: "Calling TexStorage2DMultisample is
equivalent to calling TexImage2DMultisample with the equivalently named parameters
set to the same values, except that the resulting texture has immutable
format." 

The immutable part is irrelevant for compat.

Reply:
You are correct - these comments were for a different fallback command I was trying. Then I noticed `glTexImage2DMultisample` existed in an earlier profile, so switched to using that.
Fixed the comments.